docs/third-party-packages.md

```diff
@@ -18,6 +18,13 @@ The ultimate Python library in building OAuth and OpenID Connect clients and ser
 
 An asynchronous GitHub API library. Includes [HTTPX support](https://gidgethub.readthedocs.io/en/latest/httpx.html).
 
+### VCR.py
```

Comment:
Awesome! Small nitpick though, can you put it under `RESPX` to keep alphabetical order?

Reply:
@jcugat done!